docker/Dockerfile.dev

```diff
@@ -1,4 +1,4 @@
-FROM python:3.8-slim as base
+FROM python:3.9.18-slim as base
```

Comment:
Any reason to target 3.9.18 versus 3.9?

Reply:
It's the latest 3.9, otherwise no

Reply:
I think if you use `python-3.9` it will always pull the latest Python 3.9 published version, whereas with this statement you will always target 3.9.18

Reply:
Ah ok, I'll fix that here and in the legacy domain certificate renewer when I change the slack channel

Reply:
ok, fixed that and corrected the slack channels here